### ISSUE TITLE
Custom themes 

### ISSUE THREAD

**Issue by 0x4E69676874466F78** [joeyballentine/chaiNNer#788]
I would like to make a compact theme using CSS overrides.

**Comment by RunDevelopment** [comment]
Custom themes are planned eventually, but it will be a long time until then. We currently use a UI library that uses CSS-in-JS, so CSS overrides are pretty much impossible right now. However, we are planning to eventually drop that library anyway (for performance reasons). 

So custom themes will come, but it will likely take some time since we'd have to change most of our UI code.

**Comment by 0x4E69676874466F78** [comment]
@RunDevelopment okay.
Currently you can't even load/inkeject your own css that will override resulting html layout?

**Comment by RunDevelopment** [comment]
> Currently you can't even load/inject your own css that will override resulting html layout?

No. It wouldn't be useful anyway. Because of the CSS-in-JS library, nothing has useful class names, just generated nonsense names that change every build.
